src/application/services/useAppState.ts

```diff
@@ -36,11 +37,11 @@ export const useAppState = createSharedComposable((): UseAppStateComposable => {
   /**
    * Subscribe to user changes in the App State
    */
-  AppStateController.user((prop: 'user' | 'editorTools', value: User | EditorTool[] | null) => {
+  AppStateController.user((prop: 'user' | 'userEditorTools' | 'allEditorTools', value: User | EditorTool[] | null) => {
     if (prop === 'user') {
       user.value = value as User;
     }
-    if (prop === 'editorTools') {
+    if (prop === 'userEditorTools') {
       userEditorTools.value = value as EditorTool[];
```

Comment:
why do we need this change?

Reply:
Because this is only for user tools. This naming was in @GeekaN2 PR, and it is used specifically for downloading user tools in /settings page

Reply:
it is already a subscription to the "user" state, all props here are related to user. Anyway, this change is not related to Marketplace, so it's better to clear all unrelated changes